### ISSUE TITLE
Roughness and disconnected in fresh material in Belnder 3.3.8LTS

### ISSUE THREAD

**Issue by romandesign** [AsoboStudio/glTF-Blender-IO-MSFS#255]
### Current Behavior

The moment I create a new material, without adding any images, "Occlusion Multiplier", "Roughness Multiplier" and "Metallic Multiplier" notes are created so their output is not connected to anything at all. When I add the Albedo image - it gets connected properly, and when I add the Normal image - it gets connected to Normal. However, when I add the ORM image - I see that while Occlusion and Metallic get connected to Principled BSDF, the Roughness node does not get connected to anything. So the only node connected to Roughness in BSDF is Roughness Multiplier as if the image was not there.

### Expected Behavior

When adding ORM image, all three nodes should get connected.

### Steps To Reproduce

Create new material, add Albedo, ORM and Normal images to MSFS Material.

### Environment

```markdown
- OS: Win11
- Blender: 3.3.8LTS
- glTF-Blender-IO-MSFS: 1.3.1
```

### Anything else?

![image](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/assets/9002820/5d1456d2-2de2-4106-adf2-1ddb6c9882d1)

**Comment by Safojas** [comment]
Same for me 1.3.1 + Blender 3.3.7. Comp Texture Exported with blank green channel.

**Comment by ronh991** [comment]
Yes looking at the code it seems the line 915

            self.link(nodeRoughnessScale.outputs[0], nodePrincipledBSDF.inputs[MSFS_PrincipledBSDFInputs.roughness.value])


should be

            self.link(nodeRoughness.outputs[0], nodePrincipledBSDF.inputs[MSFS_PrincipledBSDFInputs.roughness.value])
            
nodeRoughnessScale change to nodeRoughness

If you dare mess with the code.

**Comment by mamu82** [comment]
>         self.link(nodeRoughness.outputs[0], nodePrincipledBSDF.inputs[MSFS_PrincipledBSDFInputs.roughness.value])


i'm a dick at python but nodeRoughness doens't exist, look like 

 self.link(nodeMulRoughness.outputs[0], nodePrincipledBSDF.inputs[MSFS_PrincipledBSDFInputs.roughness.value])

is more appropriate, but we have to deal with a default RoughnessScale of 0.5 (why?)

so I came up with this one to bypass the roughness scale nodes at all 

self.link(nodeSeparateComp.outputs[1], nodePrincipledBSDF.inputs[MSFS_PrincipledBSDFInputs.roughness.value])

**Comment by ronh991** [comment]
@Safojas sorry my apologies I copied the wrong line info.  However that line only works if you have a Comp texture otherwise the links are from the scale nodes as shown in your picture.  Once you put a Camp texture in the material you get the connection.

should be as @mamu82 suggested 

To make the default roughness 1.0 change line 192 in msfs_material_prop_update.py

        self.msfs_roughness_factor = 1.0

And

line 26

        RoughnessFactor = 1.0

in msfs_material_props.py

Again do so if you are brave.


![DA40CGDFQ_2023 07 20_14h23m43s](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/assets/19254353/e6e53948-71ee-46dd-a49b-c3f43e98f124)

**Comment by romandesign** [comment]
OK, so what would be the final changes then, considering that materials should also with without COMP texture, or with it?

**Comment by ronh991** [comment]
The exporter program sets the links based on what you imput.  If you don't have any COMP texture then the link is directly from the two scale nodes.  If you have either or both COMP and Detailed COMP textures then the two scale links are deleted and the links from the blue multiplier nodes are connected.
![DA40_CGDFQ_2023 07 21_9h4m7s](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/assets/19254353/dbfb9108-7d1e-4c13-89bb-c21931a59199)
![DA40_CGDFQ_2023 07 21_9h2m41s](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/assets/19254353/01e9b755-332d-4501-a638-b449968f7ac3)

**Comment by Safojas** [comment]
@ronh991 i’ve made changes in code and thats works for me (node and RoughnessScale) thanks.

**Comment by y-khodja** [comment]
Hello,

It has been fixed in 1.3.2 ! 

Thank you for your report
